webviz_subsurface/_models/gruptree_model.py

```diff
@@ -0,0 +1,99 @@
+from pathlib import Path
+from typing import Callable, Dict, List, Tuple
+
+import pandas as pd
+from webviz_config.webviz_store import webvizstore
+
+from webviz_subsurface._datainput.fmu_input import scratch_ensemble
+
+
+class GruptreeModel:
+    """Class to load gruptree tables"""
```

Comment:
```suggestion
    """Facilitates loading of gruptree tables"""
```
Feeling class is a bit redundant... should we add one more sentence about what the class does? 🙂

Reply:
Yes, that was the plan. I will do that in another PR. And improve the explanation :)

Reply:
The idea was to generalize the functionality in `GroupTree` so it can be used many places (I will update the `GroupTree`-plugin in a different PR later). And maybe more grouptree-related functionality can be put here later (filtering etc). But I am open to suggestions on code structure!